<ui version="4.0" >
 <class>Turnaround</class>
 <widget class="QMainWindow" name="Turnaround" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Turnaround</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>69</y>
     <width>640</width>
     <height>388</height>
    </rect>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="fileOpenAction" />
    <addaction name="separator" />
    <addaction name="fileExitAction" />
   </widget>
   <widget class="QMenu" name="menuPlayback" >
    <property name="title" >
     <string>&amp;Playback</string>
    </property>
    <addaction name="playbackPlayAction" />
    <addaction name="playbackPauseAction" />
    <addaction name="playbackStopAction" />
   </widget>
   <widget class="QMenu" name="menu_View" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <addaction name="action_ChromaPeaks" />
    <addaction name="action_Tonnetz" />
    <addaction name="action_Key_Space" />
    <addaction name="action_Chord_Ranking" />
    <addaction name="action_Segmentation" />
   </widget>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Online_tutorial" />
    <addaction name="action_Whats_this" />
    <addaction name="action_About" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuPlayback" />
   <addaction name="menu_View" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>457</y>
     <width>640</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>30</y>
     <width>640</width>
     <height>39</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="fileOpenAction" />
   <addaction name="separator" />
   <addaction name="playbackPlayAction" />
   <addaction name="playbackPauseAction" />
   <addaction name="playbackStopAction" />
   <addaction name="separator" />
   <addaction name="action_ChromaPeaks" />
   <addaction name="action_Tonnetz" />
   <addaction name="action_Key_Space" />
   <addaction name="action_Chord_Ranking" />
   <addaction name="action_Segmentation" />
   <addaction name="separator" />
   <addaction name="action_Online_tutorial" />
   <addaction name="action_Whats_this" />
  </widget>
  <action name="fileOpenAction" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/fileopen.png</normaloff>:/icons/images/fileopen.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open</string>
   </property>
   <property name="statusTip" >
    <string>Open a song</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="fileExitAction" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/exit.png</normaloff>:/icons/images/exit.png</iconset>
   </property>
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="playbackPlayAction" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/player_play.png</normaloff>:/icons/images/player_play.png</iconset>
   </property>
   <property name="text" >
    <string>Play</string>
   </property>
   <property name="shortcut" >
    <string>Space</string>
   </property>
  </action>
  <action name="playbackStopAction" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/player_stop.png</normaloff>:/icons/images/player_stop.png</iconset>
   </property>
   <property name="text" >
    <string>Stop</string>
   </property>
   <property name="shortcut" >
    <string>Esc</string>
   </property>
  </action>
  <action name="playbackPauseAction" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/player_pause.png</normaloff>:/icons/images/player_pause.png</iconset>
   </property>
   <property name="text" >
    <string>Pause</string>
   </property>
  </action>
  <action name="action_Online_tutorial" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/help.png</normaloff>:/icons/images/help.png</iconset>
   </property>
   <property name="text" >
    <string>Online &amp;tutorial</string>
   </property>
   <property name="shortcut" >
    <string>F1</string>
   </property>
  </action>
  <action name="action_Whats_this" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/contexthelp.png</normaloff>:/icons/images/contexthelp.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;What's this</string>
   </property>
   <property name="shortcut" >
    <string>Shift+F1</string>
   </property>
  </action>
  <action name="action_About" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_ChromaPeaks" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/polarchromapeaks.svg</normaloff>:/icons/images/polarchromapeaks.svg</iconset>
   </property>
   <property name="text" >
    <string>Chroma &amp;Peaks</string>
   </property>
  </action>
  <action name="action_Tonnetz" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/tonnetz.svg</normaloff>:/icons/images/tonnetz.svg</iconset>
   </property>
   <property name="text" >
    <string>&amp;Tonnetz</string>
   </property>
  </action>
  <action name="action_Key_Space" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/keyspace.svg</normaloff>:/icons/images/keyspace.svg</iconset>
   </property>
   <property name="text" >
    <string>&amp;Key Space</string>
   </property>
  </action>
  <action name="action_Chord_Ranking" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/chordranking.svg</normaloff>:/icons/images/chordranking.svg</iconset>
   </property>
   <property name="text" >
    <string>Chord &amp;Ranking</string>
   </property>
  </action>
  <action name="action_Segmentation" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/segmentation.svg</normaloff>:/icons/images/segmentation.svg</iconset>
   </property>
   <property name="text" >
    <string>&amp;Segmentation</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections/>
</ui>
